tests/unit_tests/test_wave_06.py

```diff
@@ -33,12 +32,9 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("Electronics")
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert len(items) == 0
+    assert items == [] # would this be the best check or assert not items? 
```

Comment:
I prefer `assert items == []` vs. `assert not items`. You also don't need the `len` one if you have that. Any one of these three is basically fine, though.